src/inc/clrnt.h

```diff
@@ -789,6 +789,8 @@ typedef struct _DYNAMIC_FUNCTION_TABLE {
 //
 #ifdef _TARGET_AMD64_
 
+#define COMPACT_RUNTIME_FUNCTION        0
```

Comment:
As I understand, we need ``EndAddress``. 

The implementation of ``RtlpGetFunctionEndAddress`` for ARM/ARM64 computes the end address via multiplying the length of function (which seems to be the number of instruction) and the size of each instruction, which means that it this logic is valid only when the architecture has fixed instruction size.